deploy/platform/docker/docker-compose.agent.yaml

```diff
@@ -22,6 +22,7 @@ services:
   gateway:
     image: ${HUB}/gateway-service:${TAG}
     networks: [ sw ]
+    privileged: true
```

Comment:
Maybe you need some settings for kubernetes also?

Reply:
I think so. I will use `deploy.kubernetes` to test locally later and make this change after it is completed.